python-black.sublime-settings

```diff
@@ -1,10 +1,23 @@
 {
-    // Whether to automatically format the entire document when saving.
-    // There are three modes:
-        // - true (deprecated)
-        // - false (deprecated)
-        // - "on"
-        // - "off"
-        // - "smart": Automatic formatting is only enabled if there is a `black` section in the project's `pyproject.toml`
-    "format_on_save": "on"
+   // Whether to automatically format the entire document when saving.
+   // There are three modes:
+   //    - true (deprecated)
+   //    - false (deprecated)
+   //    - "on"
+   //    - "off"
+   //    - "smart": Automatic formatting is only enabled if there is a `black` section in the project's `pyproject.toml`
+   "format_on_save": "on",
+   // Black [OPTIONS]
```

Comment:
A description of the priority of the different configuration files should be added here.